pandas/compat/__init__.py

```diff
@@ -20,14 +19,6 @@
 IS64 = sys.maxsize > 2 ** 32
 
 
```

Comment:
The LICENSES folder doesn't contain SIX. So I guess this comment is not valid anymore.